eslint.config.js

```diff
@@ -1,28 +1,45 @@
-import js from '@eslint/js'
-import globals from 'globals'
-import reactHooks from 'eslint-plugin-react-hooks'
-import reactRefresh from 'eslint-plugin-react-refresh'
-import tseslint from 'typescript-eslint'
+import { createRequire } from "node:module";
+import js from "@eslint/js";
+import globals from "globals";
+import reactHooks from "eslint-plugin-react-hooks";
+import reactRefresh from "eslint-plugin-react-refresh";
+import eslintPluginSimpleImportSort from "eslint-plugin-simple-import-sort";
+import noRelativeImportPaths from "eslint-plugin-no-relative-import-paths";
+import tseslint from "typescript-eslint";
+
+const require = createRequire(import.meta.url);
+const packageConfig = require("./package.json");
+const packageRegex =
+  "^" + Object.keys(packageConfig.dependencies).join("|") + "$";
 
 export default tseslint.config(
-  { ignores: ['dist'] },
+  { ignores: ["dist"] },
   {
     extends: [js.configs.recommended, ...tseslint.configs.recommended],
-    files: ['**/*.{ts,tsx}'],
+    files: ["**/*.{ts,tsx}"],
     languageOptions: {
       ecmaVersion: 2020,
       globals: globals.browser,
     },
     plugins: {
-      'react-hooks': reactHooks,
-      'react-refresh': reactRefresh,
+      "react-hooks": reactHooks,
+      "react-refresh": reactRefresh,
+      "simple-import-sort": eslintPluginSimpleImportSort,
+      "no-relative-import-paths": noRelativeImportPaths,
```

Comment:
This one I'm a little less sure of, but I'm not fully against it - I'd like to hear out the reasoning a bit more!

If I'm referencing something in the same directory, isn't it easier (and clearer to read) to say  `import {Button} from './Button'` than `import {Button} from 'components/shared/Button/Button'`?

Reply:
the philosophy is to make developers more aware of the folder structure hopefully keep things organized, but I am actually going to revert this - absolute paths are not working for storybook and I was not super in love with them to begin with

Reply:
Sounds good! Thanks